libzim/wrapper.pyx

```diff
@@ -179,8 +179,8 @@ cdef public api:
         cdef map[HintKeys, uint64_t] ret;
         try:
             func = getattr(obj, method.decode('UTF-8'))
-            hintsDict = func()
-            return convertToCppHints(func())
+            hintsDict = {k: pybool(v) for k, v in func().items() if isinstance(k, Hint)}
```

Comment:
Hints values are unsigned integer. They can be something else than a bool.
Future hints may be some kind of group that libzim will use to regroup item in same cluster.

For now we are only use bool so we are ok, but we may have to change this.

Reply:
Ideally we should have cast specific to the key used. But it would always be bool for now.

Reply:
OK, so should we keep it as it is for now and defer that change to whenever we need it ? ie. in a future libzim release with additional hints?

Reply:
yes

Reply:
👍